data/adoptions.yaml

```diff
@@ -1,20 +1,18 @@
 main:
-- url: https://github.com/awslabs/tough
+- url: https://github.com/cnabio/cnab-spec/blob/cnab-security-1.0.0-ga/300-CNAB-security.md
+  tag: CNAB
+- url: https://github.com/bottlerocket-os/bottlerocket/tree/develop/sources/updater
```

Comment:
Should we change the logo, to show the bottlerocket-os logo instead of the aws logo?